src/guide.jl

```diff
@@ -47,7 +47,7 @@ const background = PanelBackground
 
 
 function render(guide::PanelBackground, theme::Gadfly.Theme,
-                aes::Gadfly.Aesthetics)
```

Comment:
Would it be easier to define a fallback function like this instead of adding so many unused parameters?
```julia
function render(guide::Guide, theme::Gadfly.Theme,
                aes::Gadfly.Aesthetics, dynamic::Bool=true)
    render(guide, theme, aes)
end
```